### PR TITLE
Testing the Fork commit 

### PR DESCRIPTION
**Pull request by royrajeshkumar**

### CODE REVIEW

**Comment by royrajeshkumar** [comment]
This commit is just to test the Fork concept in GitHub.

File: webapp/i18n/i18n.properties [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # This is the resource bundle for Management Portal
 
-#XTIT: Application name will change soon
+#XTIT: Application name will change soon!
```
Comment (rajeshroy) [inline webapp/i18n/i18n.properties]: Will see, let the admin decide name first.

File: readme.txt [hunk 2]
```diff
@@ -1,5 +1,5 @@
  #### WELCOME ####
-
+I'm simply making the fork to push my code.
```
Comment (rajeshroy) [inline readme.txt:2]: Well, happy to see you here.

File: webapp/i18n/i18n.properties [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # This is the resource bundle for Management Portal
 
-#XTIT: Application name
+#XTIT: Application name will change soon!
```
Comment (rajeshroy) [inline webapp/i18n/i18n.properties:3]: Let the Admin decide it first.

**Review by rajeshroy — CHANGES_REQUESTED** [review]
Please change the text.

**Comment by rajeshroy** [comment]
Change the comment text.

**Pull request closed without merging**